components/Starknet/modules/ROOT/pages/faq.adoc

```diff
@@ -0,0 +1,13 @@
+[id="FAQ"]
+= Frequently asked questions
+
+== What does the Sequencer do with the gas fees? (ETH or STRK)
+
+Sequencers have the role of converting STRK fees received from users into ETH to meet operational expenses. However, the frequency of these conversions can vary depending on the financial management approach of each sequencer's operator.
+At present, the only sequencer in use on Starknet is StarkWare's sequencer. It employs an ETH/STRK oracle to determine the appropriate number of STRK tokens to be charged as fees to its users.
+
+For more information on fees and how they are calculated, see xref:architecture-and-concepts:network-architecture/fee-mechanism.adoc[Gas and transaction fees].
+
+== Is there any burning mechanism?
+
+Starknet handles excess tokens from transaction fees by enabling sequencers to convert the fees paid in STRK to ETH to cover operating costs, rather than through a burning mechanism. The operator of every sequencer chooses how to manage their finances, so they determine how frequently to convert. Currently, the sequencer uses an ETH/STRK oracle to determine how many STRK tokens to charge its users as a fee.
```

Comment:
```suggestion
Starknet handles excess tokens from transaction fees by enabling sequencers to convert the fees paid in STRK to ETH to cover operating costs, rather than through a burning mechanism. The operator of every sequencer chooses how to manage their finances, so they determine how frequently to convert. 

At present, the only sequencer in use is StarkWare's sequencer. It uses an ETH/STRK oracle to determine how many STRK tokens to charge its users as a fee.
```